packages/graphql-yoga/src/plugins/request-validation/use-check-graphql-query-params.ts

```diff
@@ -18,6 +19,21 @@ export function assertInvalidParams(
       },
     });
   }
+
+  if (
+    'operationName' in params &&
+    typeof params.operationName !== 'string' &&
+    params.operationName != null
+  ) {
+    throw createGraphQLError(`Invalid operation name in the request body.`, {
+      extensions: {
+        http: {
+          status: 400,
+        },
+      },
+    });
+  }
```

Comment:
This part was missing but required in the http spec. It was caught before by chance...

Reply:
How can we ensure we don't introduce a regression? How was this cattched by chance?